src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -2363,8 +2363,9 @@ namespace CondenserLoopTowers {
 
         // PURPOSE OF THIS SUBROUTINE:
         // Initialize the simulation variables.
-
-
+        for (auto & thisTower : SimpleTower) {
+            thisTower.__AirFlowRateRatio = 0.0;
+        }
```

Comment:
This variable must need to be initialized at the beginning of the calc or init routines.  The other variables didn't have this issue, so I just need to unify the usage.